playground/AzureContainerApps/AzureContainerApps.AppHost/api-roles.module.bicep

```diff
@@ -0,0 +1,27 @@
+@description('The location for the resource(s) to be deployed.')
+param location string = resourceGroup().location
+
+param storage_outputs_name string
+
+resource identity 'Microsoft.ManagedIdentity/userAssignedIdentities@2023-01-31' = {
```

Comment:
We generate a module per container/project and that module has both the user assigned identity and role assignments for that compute resource.